src/plots/gl2d/scene2d.js

```diff
@@ -667,3 +675,18 @@ proto.hoverFormatter = function(axisName, val) {
     var axis = this[axisName];
     return Axes.tickText(axis, axis.c2l(val), 'hover').text;
 };
+
+proto.castHoverOption = function(trace, ptNumber, attr) {
```

Comment:
possible to share this with the one in [gl3d](https://github.com/plotly/plotly.js/pull/1582/commits/6bbfa652b4da40f6c060eb4ef72c97dbea98e253)? Looks nearly identical?